allensdk/brain_observatory/behavior/dprime.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import numpy as np
+from scipy.stats import norm
+
+
+def get_hit_rate(hit=None, miss=None, aborted=None, sliding_window=100):
+
+    assert len(hit) == len(miss)
+    aborted = np.logical_not(np.array(aborted, dtype=np.bool))
+    hit = np.array(hit, dtype=np.bool)[aborted]
+    miss = np.array(miss, dtype=np.bool)[aborted]
+    hit_responses = np.empty_like(hit, dtype=np.float)
+    hit_responses.fill(float('nan'))
+
+    hit_responses[hit] = 1
+    hit_responses[miss] = 0
+    hit_rate = pd.Series(hit_responses).rolling(window=sliding_window, min_periods=0).mean().values
```

Comment:
@dougollerenshaw and I agree that these last 8 lines appear to do the correct thing, but it took us a while to figure it out. Could there be some comments explaining why you need to make the NaN array and then go back and fill in the hits and misses (i.e. because the input didn't have catch trials filtered out)